web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -569,7 +569,13 @@
 
 {#if latlng && $featureFlags.loaded && $featureFlags.map}
   <div class="h-[360px]">
-    <Map mapMarkers={[{ lat: latlng.lat, lon: latlng.lng, id: asset.id }]} center={latlng} zoom={14} simplified>
+    <Map
+      mapMarkers={[{ lat: latlng.lat, lon: latlng.lng, id: asset.id }]}
+      center={latlng}
+      zoom={14}
```

Comment:
Since we have increased the zoom limit to 18 we might consider having it more zoomed in here as well. Could you test it and see if e.g. 16 looks better?

Reply:
15 looks better